ipythontutor.py

```diff
@@ -107,8 +111,28 @@ def ipythontutor(self, line, cell):
             pt_env[pass_through] = env[pass_through]
 
         request = urlencode(pt_env)
+        
         url = "{proto}://pythontutor.com/iframe-embed.html#{request}"\
               .format(request=request, **env)
+
+        # ----------------linkButton------------------------
+        # create the button
+        button = Button(description='Ouvrir dans un onglet')
+        my_output = Output()
+        
+        # if button clicked open in a new tab
+        def on_button_clicked(b):
+            with my_output:
+                webbrowser.open_new_tab(url)
+
+        # get click
+        button.on_click(on_button_clicked)
+        
+        # if true show button
+        if env['linkButton'] == 'true':
```

Comment:
je propose ici d'utiliser `from distutils.util import strtobool` pour tester cela
voir https://stackoverflow.com/questions/21732123/convert-true-false-value-read-from-file-to-boolean?lq=1

(ou de se récrire une fonction à nous)
le truc c'est que c'est bien d'être flexible, car on a vite fait d'oublier la convention et d'écrire en vitesse linkButton=True ou linkButton=1 ou linkButton=on